### PR TITLE
implemented a fairly simple class that handles the API call and return of formatted data

### PR DESCRIPTION
**Pull request by griffinstiens**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2,71 +2,70 @@
...
 
     if price == None:
         msg = f'Current Price: {currentPrice}\nOpen: {openPrice}\nPrevious Close: {prevClose}\nHigh: {highPrice}\nLow: {lowPrice}'
         await ctx.send("Loading...")
         await ctx.send(msg)
     elif price == 'current':
-        print(ticker)
-        await ctx.send("Current Price: " + str(output['c']))
+        await ctx.send("Current Price: " + currentPrice)
     elif price == 'open':
```
Comment (griffinstiens) [inline bot.py:60]: Realized that I was calling `str` on the returned data twice

File: bot.py [hunk 2]
```diff
@@ -2,71 +2,70 @@
...
 async def on_ready():
     print(f'{bot.user.name} has connected to Discord!')
 
+# When member joins, bot will DM a little message
 @bot.event
 async def on_member_join(member):
     await member.create_dm()
     await member.dm_channel.send(
-        f'Sup {member.name}...little bitch'
+        f'Sup {member.name}...Welcome to the resistance'
```
Comment (griffinstiens) [inline bot.py:32]: Figured...If someone from CRWD will maybe view this repo...Should be a little more friendly...

File: bot.py [hunk 3]
```diff
@@ -2,71 +2,70 @@
...
+async def grab_stonks(ctx, ticker: str, price = None):
+    # Instantiating Stock class
+    stocks = Stock(ticker, price, FINN_API_KEY)
+    # Returning stock data
+    stock_data = await stocks.get_data()
 
-    query = {'symbol': ticker, 'token': FINN_API_KEY}
-    res = requests.get('https://finnhub.io/api/v1/quote', params=query)
+    # Checking for an empty json response from API
+    if (stock_data == True):
```
Comment (dcargs) [inline bot.py:44]: These two are the same
```
if (stock_data == True): 
```

```
if (stock_data): 
```

Comment (griffinstiens) [reply bot.py:44]: Ye I know - Is using the short-hand typically better? I know it's more clean for sure.

**Pull request merged**